src/libs/notificationService.js

```diff
@@ -30,8 +30,12 @@ export const NotificationService = {
    * @returns {Promise<JSON>}
    */
   getActiveBanners: async () => {
-    const banners = await NotificationService.getBanners();
-    return filter(banners, {active: true});
+    try {
```

Comment:
this block fixes the "uncaught exception" error that shows up when you reload the page